### PR TITLE
BFW-689

### PR DESCRIPTION
**Pull request by espr14**

### CODE REVIEW

**Comment by espr14** [comment]
Let me know if I should squash it. Otherwise, it will be squashed during merging PR.

File: src/common/marlin_client.c [hunk 1]
```diff
@@ -144,182 +148,196 @@ void marlin_client_loop(void) {
...
 int marlin_wait_motion(uint32_t timeout) {
-    marlin_vars_t *vars = marlin_update_vars(MARLIN_VAR_MSK(MARLIN_VAR_MOTION));
-    uint32_t tick = HAL_GetTick();
+    const marlin_vars_t *vars = marlin_update_vars(MARLIN_VAR_MSK(MARLIN_VAR_MOTION));
+    const uint32_t tick = HAL_GetTick();
     while ((vars->motion == 0) && ((HAL_GetTick() - tick) < timeout))
         marlin_update_vars(MARLIN_VAR_MSK(MARLIN_VAR_MOTION));
-    return (vars->motion) ? 1 : 0;
+    return SIGN(vars->motion);
 }
```
Comment (XPila) [inline src/common/marlin_client.c:259]: I thing use SIGN is confusing, motion is bitmask, original code is more readable.

Comment (espr14) [reply src/common/marlin_client.c:259]: What about `return (bool) vars->motion`?

Comment (DRracer) [reply src/common/marlin_client.c:259]: that's not the same - please keep the original code.

**Pull request merged**